gr-zeromq/grc/zeromq_pub_sink.block.yml

```diff
@@ -35,6 +35,11 @@ parameters:
     dtype: int
     default: '-1'
     hide: ${ ('part' if hwm == -1 else 'none') }
+-   id: key
+    label: Filter Key
+    dtype: string
+    default: ''
+    hide: ${ ('part' if key == '' else 'none') }
```

Comment:
@talih0 I'm trying to test this but the `key` field is not displaying when dragging in any of the ZMQ blocks. It would probably be best to just remove the logic to hide the field and display it all the time.

Ignore the color theme, but below is what I'm seeing when building your branch with this PR. 

![zmq2](https://user-images.githubusercontent.com/11846824/74115350-b93f4b80-4b63-11ea-84e2-891c5d3351f3.png)
![zmq1](https://user-images.githubusercontent.com/11846824/74115349-b80e1e80-4b63-11ea-9259-cfa786122ba0.png)

Reply:
@devnulling Thanks for looking over the patch.
Strange that the key option doesn't appear. Looks fine on mine. 

I'll remove the auto hide option just in case and redo the PR. 

Thanks
![screenshot_pub_sink](https://user-images.githubusercontent.com/17433307/74118087-f74e6680-4b87-11ea-884f-76afcd28290c.png)

Reply:
@talih0 It would be great if you could also squash all these commits into a single commit. 

Something like below would be sufficient:

"gr-zeromq: Add optional key filtering"

Reply:
Thanks @devnulling, I made the change you suggested and squashed the commits